ACE/debian/control

```diff
@@ -257,7 +257,7 @@ Description: ACE-GUI reactor integration for Xt
 Package: libace-xtreactor-dev
 Architecture: any
 Section: libdevel
-Depends: libace-xtreactor-6.4.6 (= ${binary:Version}), libace-dev (= ${binary:Version}), libxt-dev (>= 6.4.7), ${misc:Depends}
```

Comment:
Probably some bugs in bin/make_release.py, that script should update this file

Reply:
There was a regex in `ACE/bin/make_release.py` which was unintentionally matching on the whitespace separating packages on the same line. I fixed this.